src/YoTennis/Models/Events/StartTieBreakEvent.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace YoTennis.Models.Events
+{
+    public class StartTieBreakEvent : GameEvent
```

Comment:
inconsistency: sometime you write `TieBreak` as it were two words 'tie break', sometimes you use `Tiebreak` as it were a single word 'tiebreak'. Clarify and align.